Source/Provers/SMTLib/SMTLibInteractiveTheoremProver.cs

```diff
@@ -85,6 +85,7 @@ public override async Task<Outcome> Check(string descriptiveName, VCExpr vc, Err
           processNeedsRestart = true;
         }
 
+        watch.Start();
```

Comment:
They are not (intended to be) different; it's just that using `end time - start time` to measure elapsed time is not great (because these clocks are not guaranteed to be monotonic and are not very precise anyway).  

However, there is no good reason to have them set in separate places. I cleaned things up by rewriting the code to start the stopwatch when `ProverStartTime` is set and consult it when the elapsed time was computed.